pyproject.toml

```diff
@@ -36,6 +36,7 @@ dependencies = [
     "llama-index-llms-azure-openai==0.1.5",
     "llama-index-llms-vertex==0.1.5",
     "llama-index-llms-replicate==0.1.3",
+    "llama-index-llms-ollama>=0.1.3",
```

Comment:
@harini-venkataraman pls pin down the version as done in other packages to avoid surprises with ollama

Reply:
Just saw that we have a similar operator for llama-parse
"llama-parse>=0.4.1",
Pls pin it down there also.

Reply:
@gaya3-zipstack Addressed

Reply:
@harini-venkataraman pls keep the pdm.lock file updated for these dependencies if not done already.